src/xpcc/communication/xpcc/backend/zeromq/reader.hpp

```diff
@@ -0,0 +1,177 @@
+// coding: utf-8
+/* Copyright (c) 2016, Roboterclub Aachen e. V.
```

Comment:
This should really be:
```diff
-/* Copyright (c) 2016, Roboterclub Aachen e. V.
+/* Copyright (c) 2016, Sascha Schade
+ * Copyright (c) 2017, Christopher Durand
```
We are going to [forward port these changes to modm](https://github.com/modm-io/modm/blob/develop/src/modm/communication/xpcc/backend/zeromq/connector.hpp#L2), which use the correct copyright information. So to make it a little easier for us, we are requesting this change.